.circleci/build.sh

```diff
@@ -5,12 +5,8 @@ set -e
 DOCKER_NAMESPACE=${DOCKER_NAMESPACE:-"reactioncommerce/reaction"}
 
 # if we're not on a deployment branch or a Docker related PR branch, skip the Docker build/test
-if [[ "$CIRCLE_BRANCH" != "master" && \
-      "$CIRCLE_BRANCH" != "development" && \
-      "$CIRCLE_BRANCH" != *"docker"* && \
-      "$CIRCLE_BRANCH" != "marketplace" ]]; # TODO: remove this once marketplace is merged
```

Comment:
Should I wait to merge this until after we've released 1.5.0?

Reply:
I was under the impression the `release-1.5.0` branch _was_ marketplace.  No?

Reply:
Either way, this will have no effect on whether marketplace continues to build in CI unless you merge `release-1.5.0` _back_ into marketplace.

Reply:
And we definitely want this to be part of the 1.5.0 release.  The current Docker base isn't using the current 4.x version of Node that we should be (4.8.4).